website/docs/utilities/dialog-primitive/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,30 @@
+## Content
+
+### DialogPrimitive::Header
+The header is comprised of the following: `Title icon`, `Tagline`, `Title`, and `Dismiss button`.
+#### Title Icon
+- Can be optionally displayed.
+- Purely decorative. Is the visual flourish to the title and, if displayed, the tagline.
+#### Tagline
+- Can be optionally displayed.
+- Is used to maintain context of the dialog's feature, function or flow in which it was triggered by.
+- Should be limited to a couple of words.
+#### Title
+- Is required
+- Must explain or provide context to the body's content
+#### Dismiss Button
+- Is required
+- Allows the user to close the dialog on click (or pressing the `escape` key)
```

Comment:
This is technically not correct, for the `DialogPrimitive`) so we should explain to the consumers that they have to implement for themselves this behaviour.